src/sparsezoo/utils/__init__.py

```diff
@@ -17,12 +17,14 @@
 
 import os
 
+from sparsezoo.utils.helpers import convert_to_bool
 
 BASE_API_URL = (
     os.getenv("SPARSEZOO_API_URL")
     if os.getenv("SPARSEZOO_API_URL")
     else "https://api.neuralmagic.com"
 )
+SPARSEZOO_TEST_MODE = convert_to_bool(os.getenv("SPARSEZOO_TEST_MODE"))
```

Comment:
Actually can we move this computation down into the actual function? That way the env variable is checked every time the function is called so we aren't dealing with some weird global variable caching